.github/workflows/dist-docs.yml

```diff
@@ -0,0 +1,158 @@
+name: Rebuild src/static/support/dist-docs
+
+on:
+  workflow_dispatch:
```

Comment:
This seems to be for manual triggers, is it really what you want?

Or do you want to run it on "push" (merged code) ?

Reply:
My current thinking is that we would manually trigger this workflow every time a new OVN release is made.
So, not very often. As time goes by, we may feel that can be automated and then we find a better trigger.
We could use that a cron like trigger for that too.